Makefile

```diff
@@ -3,14 +3,21 @@ include ./scripts/init.mk
 # This file contains hooks into the project configuration, test and build cycle
 # as automated steps to be executed on a workstation and in the CI/CD pipeline.
 
+install: # Install project dependencies
+	npm install
+
+build: # Build project
+	npm run build && npm run package
+
+test: # Run tests
+	npm test
+
 config: # Configure development environment
-	# TODO: Use only `make` targets that are specific to this project, e.g. you may not need to install Node.js
 	make \
-		asdf-install \
-		githooks-install \
-		nodejs-install \
-		python-install \
-		terraform-install
+		nodejs-install
```

Comment:
Could we add a call to the other two make targets `asdf-install` and  `githooks-install` to the config recipe? E.g.

```
make \
    asdf-install \
    githooks-install \
    nodejs-install
```